src/imitation/scripts/config/train_adversarial.py

```diff
@@ -129,12 +129,26 @@ def cartpole():
     discrim_net_kwargs = {"gail": {"scale": False}}
 
 
+@train_adversarial_ex.named_config
+def seals_cartpole():
+    env_name = "seals/CartPole-v0"
+    # seals and vanilla CartPole have the same expert trajectories.
+    rollout_hint = "cartpole"
+    discrim_net_kwargs = {"gail": {"scale": False}}
+
+
 @train_adversarial_ex.named_config
 def mountain_car():
     env_name = "MountainCar-v0"
     rollout_hint = "mountain_car"
 
 
+@train_adversarial_ex.named_config
+def seals_mountain_car():
+    env_name = "seals/MountainCar-v0"
+    rollout_hint = "mountain_car"  # TODO: Use seals/MountainCar-v0 rollouts.
```

Comment:
good to add issue number or username to TODO so we can get more info about it in the future. e.g. `TODO(shwang): ...`

Reply:
thanks for catching, fixed